bin/kick_off.sh

```
@@ -239,7 +247,7 @@ do
 	echo "  term_system: ${term_system}" >> ansible_vars.yml
 	system_type=`grep system_type  ansible_vars_main.yml | cut -d':' -f 2 | cut -d' ' -f 2`
 	attempts=1
-	while [ $attempts -ne $create_attempts ]
+	while [ $attempts -le $create_attempts ]
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)